### DISCUSSION TITLE
0jets events in b-jet analysis

### DISCUSSION THREAD

**Discussion by semlalisouad** [MadAnalysis/madanalysis5#168] [General]
Dear Madanalysis team,

I am having a similar issue as in  https://answers.launchpad.net/madanalysis5/+question/677992. My toolbox consists of madgraph, pythia and delphes.  I am using madanalysis to carry out the analysis. I  am forcing the light Higgs to decay to b b~ 
(h -> b b~), but I am getting about 70% of events containing 0 b-jets and about 29% of events with Nb=2. The b-tag efficiency in the Delphes card is set to 85%, and the c-jet misidentification rate is set to 25%.  Do you have any idea where these events with Nb =0 are coming from?  I believe that understanding this may help me to restore the signal. 

Many thanks,

Best
Souad

**Comment by jackaraz** [comment]
Hi @semlalisouad, there is not much to say on top of @BenjaminFuks's comments on the question that you pointed out. With delphes it's hard to say because there are a lot of things happening under the hood, so maybe your jets are butchered with some other criteria. I suggest using SFS for detector simulation, that way you can implement each effect step by step easily and understand what's happening with your simulation. You can also compare it with truth-level events as shown in [arXiv:2006.09387](https://arxiv.org/abs/2006.09387) [hep-ph].

**Comment by semlalisouad** [comment]
Hi Jack,

Many thanks for your email and suggestion. I will do that.

Best,
Souad

Le mer. 18 janv. 2023 à 15:00, Jack Y. Araz ***@***.***> a
écrit :

> Hi @semlalisouad <https://github.com/semlalisouad>, there is not much to
> say on top of @BenjaminFuks <https://github.com/BenjaminFuks>'s comments
> on the question that you pointed out. With delphes it's hard to say because
> there are a lot of things happening under the hood, so maybe your jets are
> butchered with some other criteria. I suggest using SFS for detector
> simulation, that way you can implement each effect step by step easily and
> understand what's happening with your simulation. You can also compare it
> with truth-level events as shown in arXiv:2006.09387
> <https://arxiv.org/abs/2006.09387> [hep-ph].
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/MadAnalysis/discussions/168#discussioncomment-4718327>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP3TVZ4GNID5RVGP6ZIS7CTWTAASJANCNFSM6AAAAAAT7FYCAU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
Souad SEMLALI
